env.example

```diff
@@ -1,3 +1 @@
-REACT_APP_MAPBOX_TOKEN='YOUR_MAPBOX_API_TOKEN'
-REACT_APP_BACKEND_URL='URl_To_BACK_API'
-REACT_APP_INSTANCE_TITLE="INSTANCE_NAME"
\ No newline at end of file
+REACT_APP_BACKEND_URL='URl_To_BACK_API'
```

Comment:
Could you put a sane default instead ? Maybe something like `http://localhost:3001`
`3001` will be the default `back` port now.

Reply:
Also updated in the back repo
https://github.com/datatlas-erasme/back/pull/13